docs/ExtensionPoints/ProductSubscription/README.md

```diff
@@ -0,0 +1,309 @@
+
+#### ProductSubscription Add
+
+| Name      | Type     | Description                            |
+| --------- | -------- | -------------------------------------- |
+| productId | `string` | The id of the current product.         |
+| variantId | `string` | The id of the current product variant. |
+
+#### ProductSubscription Create
+
+| Name      | Type     | Description                            |
+| --------- | -------- | -------------------------------------- |
+| productId | `string` | The id of the current product.         |
+| variantId | `string` | The id of the current product variant. |
+
+#### ProductSubscription Edit
+
+| Name           | Type     | Description                            |
+| -------------- | -------- | -------------------------------------- |
+| productId      | `string` | The id of the current product.         |
+| variantId      | `string` | The id of the current product variant. |
+| sellingGroupId | `string` | The id of the current selling group.   |
+
+#### ProductSubscription Remove
+
+| Name           | Type       | Description                              |
+| -------------- | ---------- | ---------------------------------------- |
+| productId      | `string`   | The id of the current product.           |
+| variantId      | `string`   | The id of the current product variant.   |
+| variantIds     | `string[]` | The ids of the current product variants. |
+| sellingGroupId | `string`   | The id of the current selling group.     |
+
+### Container API
+
+See [Container](../../Utilities/Container.md) for more info about how to customize your extension’s container component using the Container API.
+
+There are two different container types for the Product subscription extension, depending on the extension mode:
+
+#### Add, Remove
+
+**App modal container**: A smaller overlay typically used for editing an existing configuration or confirming a change. In the app modal container, primary and secondary action buttons are responsible for triggering a custom behavior, such as performing a resource update.
+
+#### Create, Edit
+
+**App overlay container**: A full-screen container designed for complex tasks, like configuring a subscription for the first time.
+
+#### Add, Remove (Modal container)
+
+| Name               | Type       | Description                                                                            | Required |
+| ------------------ | ---------- | -------------------------------------------------------------------------------------- | -------- |
+| close              | `function` | Closes the container and the extension                                                 |          |
+| done               | `function` | Notify Shopify Admin that the extension workflow is complete and data has been updated |          |
+| setPrimaryAction   | `function` | Sets the primary action content and callback when the action is clicked                |          |
```

Comment:
Is it set when clicked? Or when the extension is loaded/extension mode action is clicked?

Reply:
Yeah I think you're right the sentence could be interpreted in 2 ways. Maybe "Sets the primary action content and action pressed callback". Something like that?

Reply:
Now that disabled is added to this, maybe we should simplify this to `Sets the primary action`, then add a table below for the actions prop.